### PR TITLE
additional tools

### PR DESCRIPTION
**Pull request by patrickuhlmann**
add cevimail and ceviwiki

### CODE REVIEW

File: assets/tools-frontpage/Source/kursfinder.png
Comment (maede97) [inline assets/tools-frontpage/Source/kursfinder.png:1]: can we only store a link to this and not the screenshot of the page?

File: assets/tools-frontpage/Source/ceviemail.png
Comment (maede97) [inline assets/tools-frontpage/Source/ceviemail.png:1]: can we only store a link to this and not the screenshot of the page?

**Review by maede97 — CHANGES_REQUESTED** [review]
two remarks for the clipboard images.

File: assets/tools-frontpage/Source/kursfinder.png
Comment (patrickuhlmann) [reply assets/tools-frontpage/Source/kursfinder.png:1]: ok fine

File: assets/tools-frontpage/Source/ceviemail.png
Comment (patrickuhlmann) [reply assets/tools-frontpage/Source/ceviemail.png:1]: ok fine

**Review by wp99cp — APPROVED** [review]
looks fine to me now

**Comment by maede97** [comment]
Why do we need to store the pictures now twice? once in source folder, once outside?
isn't it enough to store them where the image is actually needed and in the source folder we only store the txt file?

**Comment by patrickuhlmann** [comment]
I edited the images 

> Why do we need to store the pictures now twice? once in source folder, once outside? isn't it enough to store them where the image is actually needed and in the source folder we only store the txt file?

I edited the images that we actually use (made them quadratic), the original images can be used for reverse image search or also if other editing (e. g. adding cevi logo etc.) should take place. Images can disappear from openclipart. The idea of the original webpage screenshot was also to have some proof that the images are actually from openclipart and see the name of the author etc.Just to have something to work in case there would be an Abmahnung or something similar. From personal experience I know that sometimes the internet can be quite short lived and links soon won't work anymore leaving people with no clue what was actually there at the time we took it.

**Comment by patrickuhlmann** [comment]
> two remarks for the clipboard images.

Can I merge it?

**Comment by maede97** [comment]
Sorry, did not see your comment before. Merge it! :-)

**Review by maede97 — APPROVED** [review]

**Pull request merged**